src/main/kotlin/io/snyk/plugin/net/CliConfigService.kt

```diff
@@ -25,13 +25,25 @@ data class CliConfigSettings(
     val localCodeEngine: LocalCodeEngine,
 
     @SerializedName("reportFalsePositivesEnabled")
-    val reportFalsePositivesEnabled: Boolean
+    val reportFalsePositivesEnabled: Boolean,
+
+    @SerializedName("code")
```

Comment:
I suppose this is the HTTP Code and HTTP message?

I don't think that technical information should be part of a return data class/object. It's exposing technical details that don't need to be exposed. You're mixing normal flow with error flow - which I would advise against. 

It would in my opinion be better to have only the fields needed in the data class, and have error/exception handling done separately, setting the config accordingly.